.github/workflows/release.yaml

```diff
@@ -6,23 +6,6 @@ on:
       - "release/*"
 
 jobs:
-  release:
-    name: "release"
-    runs-on: "ubuntu-latest"
-
-    strategy:
-      # Run each build to completion, regardless of if any have failed
-      fail-fast: false
-    steps:
-      - name: create-release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-        with:
-          if-no-files-found: error
-          name: build-linux
-          path: ucm-linux.tar.gz
-  
   release:
```

Comment:
Here's the one I think we actually want.

Reply:
This one uses softprops/actions-gh-release as opposed to the canonical github actions/create-release. I was hoping the softprops readme would explain why both exist, but it doesn't seem to. 🤷 there's not really any way to test these things other than merging and see what happens...